plaso/cli/helpers/analysis_plugins.py

```diff
@@ -61,6 +61,7 @@ def ParseOptions(cls, options, configuration_object):
 
     Raises:
       BadConfigObject: when the configuration object is of the wrong type.
+      BadConfigOption: when a non-existing analysis plugins are specified.
```

Comment:
typo, should be "plugin" not "plugins"

Reply:
I'll remove the `a` to match the error string in the code.